lib/eml2html/converter.rb

```diff
@@ -55,6 +56,33 @@ def save_zip(options = {})
       end
     end
 
+    def get_attachments(include_in_doc=false)
+      attachments = []
+      if include_in_doc
+        attachments = @attachments
+      else
+        @attachments.each do |a|
+          unless @cids_in_body.includes(a.cid)
+            attachments.push(a)
+          end
+        end
+      end
+      attachments
+    end
+
+    def get_html
+      html_body
+    end
+
+    def get_attachment(file_name)
```

Comment:
Same as above, don't call it `get_attachment`. `attachment_by_name` would be more appropriate.
```ruby
@attachments.find{|a| a.name == file_name }&.content
```

Reply:
done